src/plugins/next/README.md

```diff
@@ -19,15 +19,15 @@ or `devDependencies`:
       "app/**/{error,layout,loading,not-found,page,template}.{js,jsx,ts,tsx}",
       "instrumentation.{js,ts}",
       "app/{manifest,sitemap,robots}.{js,ts}",
-      "app/**/{icon,apple-icon}-image.{js,ts,tsx}",
+      "app/**/{icon,apple-icon}.{js,ts,tsx}",
```

Comment:
This change is due to me regenerating the docs.

Reply:
Yeah I just merged something else, forgot to add that. Thanks!